pkg/event/schedule_handler.go

```diff
@@ -0,0 +1,63 @@
+package event
+
+import (
+	"go.uber.org/zap"
+)
+
+// Trend means the operation trend Trend with num will show the event  mean.
+// for example, Increase function a with number 2 means you wanna the instance a increase to 2.
+type Trend string
+
+const (
+	Increase       Trend  = "Increase"
+	Decrease       Trend  = "Decrease"
+	ScheduleSource Source = "source"
+)
```

Comment:
I prefer to name variables as `IncreaseXXX` and `DecreaseXXX` to have a stronger semantics